bootstrap.ml

```diff
@@ -233,13 +237,32 @@ let cleanup ~keep_ml_file =
   with _ ->
     ()
 
+let readdir path =
+  Sys.readdir path
+  |> Array.to_list
+  |> List.map ~f:(Filename.concat path)
+
+let find =
+  let rec loop acc = function
+    | [] -> List.sort ~cmp:String.compare acc
+    | p :: ps ->
+      let (files, dirs) =
+        readdir p
+        |> List.partition ~f:Sys.is_directory
+      in
+      loop (ps @ dirs) (List.rev_append files acc)
```

Comment:
Shouldn't this be `loop (List.rev_append files acc) (ps @ dirs)` ?

Reply:
Fixed. Thanks.